spec/mollie/client_spec.rb

```diff
@@ -62,6 +62,42 @@
     end
   end
 
+  context 'methods' do
+    it 'returns a hash with methods' do
+      VCR.use_cassette('get methods list') do
+        client = Mollie::Client.new(api_key)
+        response = client.methods
+
+        expect(response['totalCount']).to eql 8
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.